utils/rpms/daos.spec

```diff
@@ -168,7 +168,7 @@ Requires: libpmemobj = 1.11.0-3%{?dist}
 Requires: mercury = %{mercury_version}
 Requires(post): /sbin/ldconfig
 Requires(postun): /sbin/ldconfig
-Requires: libfabric >= %{libfabric_version}
+Requires: libfabric = %{libfabric_version}
```

Comment:
This (`=`) is really undesirable (and unnecessary).  This restricts our ability to release a libfabric update in the future without having to go through a whole release process (i.e. package rebuild, QA, SDL, etc.) cycle for daos.  If we leave this as `>` and find a need in the future for a new libfabric, we can release just the new libfabric (after internal verification with the already released version of daos) and not have to go through the whole release process for a new daos that does nothing new except change it's libfabric requirement.